### PR TITLE
chore(build): remove the auto-generated Java code by thrift compiler

### PR DESCRIPTION
**Pull request by shenxingwuying**
### What problem does this PR solve? 

Many auto codes by thrift generated by thrift, that's not necessary.
Byt It depends a higher thrift version, it's download url has invalid, an issue :
https://github.com/apache/incubator-pegasus/issues/1038

### CODE REVIEW

File: java-client/idl/recompile_thrift.sh [hunk 1]
```diff
@@ -19,21 +19,28 @@
...
-# Note: by default the thrift will be installed in /usr/local. If you'd like to install it somewhere else,
-# you'd better provide install path with option `--prefix=xxx'.
- 
+function GenThriftTool() {
+  wget http://mirrors.tuna.tsinghua.edu.cn/apache/thrift/0.11.0/thrift-0.11.0.tar.gz
+  tar xvf thrift-0.11.0.tar.gz
+  cd thrift-0.11.0
+  ./configure --with-csharp=no --with-java=no --with-python=no --with-erlang=no --with-perl=no \
+      --with-php=no --with-ruby=no --with-haskell=no --with-php_extension=no --with-rs=no --with-go=no
+  make
```
Comment (acelyc111) [inline java-client/idl/recompile_thrift.sh:28]: Better to add ` -j $(($(nproc)/2+1))` to make faster.

File: java-client/idl/recompile_thrift.sh [hunk 2]
```diff
@@ -19,21 +19,28 @@
...
+  sudo make install
```
Comment (acelyc111) [inline java-client/idl/recompile_thrift.sh:29]: It's suggest to install packages in Dockerfile.

File: java-client/idl/recompile_thrift.sh (hunk 1 above)
Comment (acelyc111) [reply java-client/idl/recompile_thrift.sh]: leave some CPU resource for other purpose

File: java-client/idl/recompile_thrift.sh [hunk 3]
```diff
@@ -19,21 +19,33 @@
...
-#       --with-php=no --with-ruby=no --with-haskell=no --with-php_extension=no --with-rs=no --with-go=no
-#   make
-#   sudo make install
-# 
-# Note: by default the thrift will be installed in /usr/local. If you'd like to install it somewhere else,
-# you'd better provide install path with option `--prefix=xxx'.
- 
+function GenThriftTool() {
+    set -e
+    wget --progress=dot:giga https://github.com/apache/thrift/archive/refs/tags/0.11.0.tar.gz -O thrift-0.11.0.tar.gz
```
Comment (acelyc111) [inline java-client/idl/recompile_thrift.sh:24]: Why we use thrift-0.11.0 in Java and 0.9.3 in all the other places, there must be some historical reason.
I can remember a little there is a compatiable reason in Java client that we shaded thrift 0.11.0, but not sure.
Any body could please clarify it? Many thanks!

File: java-client/idl/recompile_thrift.sh [hunk 4]
```diff
@@ -19,21 +19,33 @@
...
+    tar xzf thrift-0.11.0.tar.gz
+    pushd thrift-0.11.0
+    ./bootstrap.sh
+    ./configure --with-csharp=no --with-java=no --with-python=no --with-erlang=no --with-perl=no \
+        --with-php=no --with-ruby=no --with-haskell=no --with-php_extension=no --with-rs=no --with-go=no
```
[the remark this answers is not in the document]
Comment (shenxingwuying) [reply java-client/idl/recompile_thrift.sh]: > We only need the execute binary? You can set `--enable_libs=no` instead.

The command copy from history' comments, as left side.
I try it.

File: java-client/build.sh [hunk 5]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (shenxingwuying) [reply java-client/build.sh:1]: > Add license header

ok

File: java-client/idl/recompile_thrift.sh (hunk 3 above)
Comment (shenxingwuying) [reply java-client/idl/recompile_thrift.sh:24]: > Why we use thrift-0.11.0 in Java and 0.9.3 in all the other places, there must be some historical reason. I can remember a little there is a compatiable reason in Java client that we shaded thrift 0.11.0, but not sure. Any body could please clarify it? Many thanks!

I don't know the history's problems., and the problems not just thrift version. One more thing,
in fact, I think the thrift proto files (for interface) should be one shared by cpp and java and others. But the proto files has very large diffenences between cpp and java.

File: java-client/build.sh [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
```
Comment (acelyc111) [inline java-client/build.sh:1]: There are too many shell scripts in the project, and we are going to clean them up, it's not suggest to add more shell scripts now. You can describe the build/install/test commands in README instead.

Comment (shenxingwuying) [reply java-client/build.sh:1]: ok

**Review by acelyc111 — DISMISSED** [review]
Great!

**Review by acelyc111 — APPROVED** [review]

File: docker/pegasus-build-env/centos6/Dockerfile [hunk 7]
```diff
@@ -20,16 +20,16 @@ FROM centos:6.10
 LABEL maintainer=jiashuo
 
 # defaut repo url hasn't been supported, use thirdparty repo url
-RUN curl https://www.getpagespeed.com/files/centos6-eol.repo --output /etc/yum.repos.d/CentOS-Base.repo
-RUN curl https://www.getpagespeed.com/files/centos6-epel-eol.repo --output /etc/yum.repos.d/epel.repo
-RUN curl https://www.getpagespeed.com/files/centos6-scl-eol.repo --output /etc/yum.repos.d/CentOS-SCLo-scl.repo
-RUN curl https://www.getpagespeed.com/files/centos6-scl-rh-eol.repo --output /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo
+RUN wget --progress=dot:giga https://www.getpagespeed.com/files/centos6-eol.repo -O /etc/yum.repos.d/CentOS-Base.repo
```
Comment (foreverneverer) [inline docker/pegasus-build-env/centos6/Dockerfile:23]: it seems not have `wget` tools for the new centos env

File: java-client/src/main/java/org/apache/pegasus/apps/batch_get_request.java [hunk 8]
```diff
@@ -1,469 +0,0 @@
-/*
```
Comment (foreverneverer) [inline java-client/src/main/java/org/apache/pegasus/apps/batch_get_request.java:1]: These auto-generated file should add into `.gitignore`, other same

Comment (acelyc111) [reply java-client/src/main/java/org/apache/pegasus/apps/batch_get_request.java:1]: +1

File: docker/pegasus-build-env/centos6/Dockerfile (hunk 7 above)
Comment (shenxingwuying) [reply docker/pegasus-build-env/centos6/Dockerfile:23]: 1. Docker lint will warn both curl and wget used together. So avoid this should use only one.
2. curl get the file thrift-0.11.0.tgz, I failed. So I fix curl before to wget.
3. If wget is not exist. Keey curl and ignore the warnings in docker lint?

Comment (shenxingwuying) [reply docker/pegasus-build-env/centos6/Dockerfile:23]: -:81 DL4001 warning: Either use Wget or Curl but not both

File: java-client/src/main/java/org/apache/pegasus/apps/batch_get_request.java (hunk 8 above)
Comment (shenxingwuying) [reply java-client/src/main/java/org/apache/pegasus/apps/batch_get_request.java:1]: done

File: docker/pegasus-build-env/centos6/Dockerfile (hunk 7 above)
Comment (shenxingwuying) [reply docker/pegasus-build-env/centos6/Dockerfile:23]: DONE

Comment (foreverneverer) [reply docker/pegasus-build-env/centos6/Dockerfile]: > 2\. curl get the file thrift-0.11.0.tgz, I failed. So I fix curl before to wget.

thrift can be fetched by `wget`, but the centos6-epel-eol.repo...., should use `curl`

Or you can test, whether `wget` exist or not for the new centos6 env. I remember it don't have

**Pull request merged**